ourrevolution/settings.py

```diff
@@ -247,6 +247,7 @@
 MEDIA_URL = '/media/' if os.environ.get('env', 'development') != 'production' else 'https://s3.amazonaws.com/our-revolution-dot-com/'
 
 ORGANIZING_GUIDES_URL = '/docs/organizing-guides/'
+ORGANIZING_DOCS_URL = '/docs'
```

Comment:
Mostly good, but I suggest to add trailing slash, and suggest alpha sort the variables here (docs above guides).

Reply:
@cjmabry actually this breaks logic without a trailing slash. When current page is docs page, menu item not showing up as active.